projects/README.md

```diff
@@ -2,16 +2,14 @@
 
 |project|license|description|version|repository|patch/diff|
 |-------|-------|-----------|-------|----------|----------|
-|[FFmpeg](https://www.ffmpeg.org/ 'FFmpeg website')|[LGPL](https://www.ffmpeg.org/legal.html 'Lesser GPL v2.1')|complete, cross-platform solution to record, convert and stream audio and video|2.6.2|[repo](https://github.com/distributepro/FFmpeg 'forked FFmpeg repo on github')|[diff](https://github.com/distributepro/FFmpeg/compare/FFmpeg:n2.6.2...xp2.6.2 'patch/diff')|
 |[GLEW](http://glew.sourceforge.net/ 'The OpenGL Extension Wrangler Library')|[open](http://glew.sourceforge.net/credits.html 'Modified BSD, Mesa 3-D (MIT), and Khronos (MIT)')|The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source C/C++ extension loading library.|1.13.0|[repo](https://github.com/nigels-com/glew 'GLEW repo on github')|none|
-|[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.16|none|[diff](https://github.com/distributePro/ovsrpro/blob/master/patches/hdf5.patch 'patch/diff')|
+|[HDF-GROUP](http://www.hdfgroup.org 'The HDF Group')|[open](https://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING 'HDF5 License')|HDF5 is a data model, library, and file format for storing and managing data|1.8.16|none|[patch](../patches/hdf5.patch 'patch')|
```

Comment:
you can look at the README.md in the fixReadmePatch branch https://github.com/smanders/ovsrpro/blob/fixReadmePatch/projects/README.md and see that this is better -- it's not hard-coded to the master branch version of the patch, the link points to the branches' version of the patch

externpro will generate the README this way in the next release -- here's the issue https://github.com/smanders/externpro/issues/162 and the commit https://github.com/smanders/externpro/commit/a09473fbd5e753fa93af5520b350df097aca44cd that makes it happen